docs/stack/about/index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: About Redis Stack
+linkTitle: About
+description: Learn more about Redis Stack
+weight: 1
+---
+
+Redis Stack is an improved starting point for working with Redis. We've bundled together the best of the technology that we have to offer into an easy 
+to use package. Redis Stack extends the core capabilities of Redis OSS and provides a complete developer experience for debugging and more.
+
+In addition to all of the features of Redis OSS, Redis Stack supports:
+
+* Probabilistic data structures
```

Comment:
Should these link to JSON + friends, instead of "in stack" below?

Reply:
@chayim individual module sections are already linked elsewhere